src/main/java/com/jfrog/ide/idea/configuration/ServerConfigImpl.java

```diff
@@ -497,7 +497,7 @@ public boolean readConnectionDetailsFromJfrogCli() throws IOException {
         String passwordCli = cliServerConfig.getPassword();
         String accessToken = cliServerConfig.getAccessToken();
 
-        if ((isAnyBlank(usernameCli, passwordCli) && isBlank(accessToken)) || isAllBlank(platformUrlCLi, xrayUrlCli, artifactoryUrlCli)) {
+        if ((isAnyBlank(usernameCli, passwordCli) && isBlank(accessToken)) || isAnyBlank(xrayUrlCli, artifactoryUrlCli)) {
```

Comment:
What about the platformUrlCLi?
Maybe we should not read it at all?

If we keep it, we probably should rename it to platformUrlCli .